Ihor/VanGoux/scss/mixins.scss

```diff
@@ -77,154 +51,115 @@
-}
-
-@mixin h1_white {
-    font-family: Mulish;
-    font-size: 30px;
-    font-weight: normal;
-    font-stretch: normal;
-    font-style: normal;
-    line-height: 2;
-    letter-spacing: normal;
-    color: $white;
-}
-
-@mixin divider_white {
-    width: 80px;
-    height: 2px;
-    box-shadow: 0 2px 0 0 #6a9eff;
-    background-color: $white;
-    position: absolute;
-    right:1120px;
-}
-@mixin divider_white_mobile {
-    width: 80px;
-    height: 2px;
-    box-shadow: 0 2px 0 0 #6a9eff;
-    background-color: $white;
-    position: static;
-    right: 0;
+    }
+    .person_text {
+        font-size: 16px;
+        font-weight: normal;
+        line-height: 1.88;
+        color: $dark_blue;
+    }
 }
 
-@mixin divider {
-    width: 80px;
-    height: 2px;
-    box-shadow: 0 2px 0 0 #d1e1fe;
-    background-color: $blue;
-    position: absolute;
-    right:1120px;
-}
-@mixin divider_mobile {
-    width: 80px;
-    height: 2px;
-    box-shadow: 0 2px 0 0 #d1e1fe;
-    background-color: $blue;
-    position: static;
-    right: 0;
-}
 
-@mixin feattab {
+@mixin featcard {
     width: 320px;
-            height: 290px;
-        
-            &_head {
-                display: flex;
-                justify-content: left;
-                align-items: center;
-                background-color:$white_2;
+    height: 290px;
+    &_head {
+        display: flex;
+        justify-content: left;
+        align-items: center;
+        background-color:$white_2;
             
-                &_text {
-                    
-                    padding: 20px;
-                    font-family: Mulish;
-                    font-size: 20px;
-                    font-weight: 600;
-                    font-stretch: normal;
-                    font-style: normal;
-                    line-height: 1.5;
-                    letter-spacing: normal;
-                    color: $dark_blue;
-                }
-            }
-            &_text {
-                box-shadow: 0 20px 60px 0 rgba(49, 69, 244, 0.1);
-                background-color: $white;
-                font-family: Mulish;
-                font-size: 16px;
-                font-weight: normal;
-                font-stretch: normal;
-                font-style: normal;
-                line-height: 1.88;
-                letter-spacing: normal;
-                color: $dark_blue;
-                padding: 40px;
-            }
+        &_text {
+            padding: 20px;
+            font-size: 20px;
+            font-weight: 600;
+            line-height: 1.5;
+            color: $dark_blue;
+        }
+    }
+    &_text {
+        box-shadow: 0 20px 60px 0 rgba(49, 69, 244, 0.1);
+        background-color: $white;
+        font-size: 16px;
+        font-weight: normal;
+        line-height: 1.88;
+        color: $dark_blue;
+        padding: 40px;
+    }
 }
-@mixin feattab_mobile {
+
+@mixin featcard_mobile {
     width: 100%;
-    height: 200px;
+    max-height: 200px;
```

Comment:
Don't set max-height on blocks where content might be changed in feature. If you want to set height use min-height instead.